### ISSUE TITLE
Make a Pretty Terminal

### ISSUE THREAD

**Issue by peblair** [crew/crewcloud-dashboard#1]
Something for the to-do list (not a priority, but I thought I'd add it anyway):

We should have a nice terminal on the web interface. By this, I mean two things:
1. Colors. This will require some finagling, however, since, from what I can see, OpenStack appears to not send any type of metadata (with things like color) on VNC REST API queries. I'll be looking into a solution for this in the coming days and will update the issue accordingly.
2. Select-able text. Why? Because "why not?". If we want a sleek interface, we might as well go all Web 2.0 on it and follow the trend of implementing some ~~overly bulky and convoluted~~ nice JS-backed web application which runs the terminal interface.

tl;dr We should have a web VNC client which doesn't make users feel that SSH is the only option for interfacing with their VM.
